Dockerfile

```diff
@@ -11,10 +12,21 @@ ENV NPM_CONFIG_LOGLEVEL ${LOGLEVEL}
 # Install base dependencies
 RUN npm -g install npm@8.x.x
 
-WORKDIR /morello-api
+COPY package*.json ./
+COPY tsconfig.json ./
 
+RUN npm install
 COPY . .
+RUN npm run build
+
+# Service
+FROM node:16-alpine as service
+
+WORKDIR /morello-api
+COPY package*.json ./
 RUN npm install --production
+COPY --from=builder /morello-api/build .
+COPY --from=builder /morello-api/config ./config
 
 EXPOSE 80
```

Comment:
Is this the port your application runs on?

Reply:
No, to be fair this is something that should have been loaded forom production.json, just re-added with port 80